buildlib/azure-pipelines-release.yml

```diff
@@ -17,15 +17,15 @@ stages:
     jobs:
       - job: release
         displayName: build tarball and source rpm
-        container: fedora
+        container: fedora:2
```

Comment:
Please change it on line 12 (`mage: ucfconsort.azurecr.io/ucx/fedora:1` --> `mage: ucfconsort.azurecr.io/ucx/fedora:2`). Here you are referencing it by the name. Same for the regular pipeline.